packages/kitten-scientists/source/ui/InternalsUi.ts

```diff
@@ -41,6 +42,16 @@ export class InternalsUi extends SettingsPanel<EngineSettings> {
                 },
               }),
             ),
+            new SettingListItem(host, host.engine.i18n("ui.ksColumn"), settings.ksColumn, {
+              onCheck: () => {
+                localStorage.setItem("ks.Column", "#ksColumn");
```

Comment:
You should not need to remember anything in localStorage here. The information is already preserved in the settings.